sources/packages/api/src/database/migrations/1628287630225-OfferingIntensity.ts

```diff
@@ -11,8 +11,6 @@ export class OfferingIntensity1628287630225 implements MigrationInterface {
   }
 
   public async down(queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query(
-      getSQLFileData("Remove-offering-intensity.sql", DIR),
-    );
+    await queryRunner.query(getSQLFileData("Drop-offering-intensity.sql", DIR));
```

Comment:
Is there a reason why one is using remove and the other is using drop?

Reply:
remove is for a column, the drop is to drop the whole enum